tools/clean-repo

```diff
@@ -8,4 +9,4 @@
 #     chmod +x .git/hooks/post-checkout
 
 cd "$(dirname "$0")/.."
-find . -name '*.pyc' -delete
+find . -name "*.pyc" -exec rm -rf {} \;
```

Comment:
This can probably be `rm -f` (without the r) since .pyc files should always be files